plugins/scaffolder-node/src/actions/types.ts

```diff
@@ -76,6 +77,11 @@ export type ActionContext<
    * Optional value of each invocation
    */
   each?: JsonObject;
+
+  /**
+   * Get the credentials for the current request
+   */
+  getInitiatorCredentials(): Promise<BackstageCredentials>;
```

Comment:
Bit of a random nit, but thinking it's best to move this up to the other methods, underneath the `createTemporaryDirectory` or thereabouts

Reply:
Done